app/assets/geojsons/V_0.geojson

```diff
@@ -638,31 +638,33 @@
                 "level_name": "0"
             },
             "geometry": {
-                "type": "LineString",
+                "type": "Polygon",
```

Comment:
This was the problem: Using LineString the coordinates array was just an array of coordinates but it should be a Polygon wich has as coordinates an array with another array with the coordinates.